lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @use '../../scss-shared/nx-header-variables';
+
+.nx-toast-container {
+  grid-area: content;
+  position: static;
+  justify-self:end;
+  margin-right: 8px;
+  width: 500px;
+  z-index: 1060;
+  overflow: hidden;
+  pointer-events: none;
+}
+
+// in legacy layouts where page scrolling is enabled, position: fixed helps keep
+// NxToastContainer in place. Top refers to the set height of the the header
+.nx-html--page-scrolling {
+  .nx-toast-container {
+    position: fixed;
+    top: nx-header-variables.$main-header-height;
+  }
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+  right: 0;
+  opacity: 1;
+  overflow: hidden;
+  margin-top: var(--nx-spacing-2x);
+
+  &.nx-toast--closing {
+    animation-name: animationOut;
+  }
+
+  .nx-alert {
+    margin: 0;
+    pointer-events:auto;
```

Comment:
```suggestion
    pointer-events: auto;
```

Reply:
spacing fixed on scss for point-events here: [3deee00](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/3deee004bfc1d2460397ade2edc819b2cbbc6412)